kpconv_torch/io/las.py

```diff
@@ -0,0 +1,78 @@
+import laspy
+import numpy as np
+
+
+def read_las_laz(filepath):
+    """Takes a file path pointing on a 3D point .las or .laz file and returns the points,
+    the associated colors and the associated classes.
+
+    Parameters
+    ----------
+    filepath: path to a 3D points file with .laz or .las format
+
+    Colors are original encoded on 2 bytes
+    (cf. https://www.asprs.org/a/society/committees/standards/asprs_las_spec_v13.pdf), that is on a uint16.
+    They are converted (divided by 256) when the las or the laz file is read by this function.
+
+    Returns
+    -------
+    points: 2D np.array with type float32
+    colors: 2D np.array with type uint8
+    labels: 1D np.array with type int32
+    """
+    data = laspy.read(filepath)
+    points = np.vstack([data.x, data.y, data.z]).transpose().astype(np.float32)
+    colors = (
+        np.vstack([data.red / 256, data.green / 256, data.blue / 256]).transpose().astype(np.uint8)
+    )
+    labels = np.array(data.classification).astype(np.int32)
```

Comment:
Quite the same remark than for `.xyz`: we will have to think about the case of `.ply` files without classification data. (That's not introduced by your PR, we can postpone the check for a further PR.)